spaces/S000046/properties/P000021.md

```diff
@@ -7,5 +7,4 @@ refs:
   name: Counterexamples in Topology
 ---
 
-Asserted in the General Reference Chart for space #54 in
-{{doi:10.1007/978-1-4612-6290-9_6}}.
+For each point $x$ there is another point $y$ topologically indistinguishable from it.  Then $y$ is a limit point of any set containing $x$.
```

Comment:
This seems deserving of a utility property/theorem: every space that has the property that every point is topologically indistinguishable from another point is weakly countably compact (in fact, every non-empty set has a limit point, not just infinite).

What to name this property? Anti-Kolomorgov or Anti-$T_0$? I don't think it fits the [Bankston definition for "anti"](https://epublications.marquette.edu/cgi/viewcontent.cgi?article=1162&context=mscs_fac) though (I guess that would be equivalent to indiscrete). 

Maybe "contra-$T_0$"?

Reply:
I had the same first impression that it could be a more general utility property.  But after further thought, it's not a very common occurrence among useful examples.  Most interesting examples are $T_0$ anyway, and if not, their properties can often be deduced from their Kolmogorov quotient.  We could discuss further (in zoom meeting?), but I feel it's better not to add this to pi-base for now.  There are so many other properties that deserve attention instead.

Reply:
I don't see a pressing need to discuss, makes sense to me.